3-reveal/migrations/5-IRS/2-Zambia-2019/deploy/create_structure_geo_hierarchy.psql

```diff
@@ -22,16 +22,16 @@ BEGIN
         jurisdictions.geographic_level AS jurisdiction_depth,
         geo_jurisdictions.id AS geo_jurisdiction_id,
         geo_jurisdictions.geographic_level AS geo_jurisdiction_depth,
-        ST_WITHIN(locations.geometry, geo_jurisdictions.geometry) AS geo_strict_within
-    INTO STRICT geo_record
+        public.ST_WITHIN(locations.geometry, geo_jurisdictions.geometry) AS geo_strict_within
+    INTO geo_record
     FROM locations
     LEFT JOIN jurisdictions
         AS geo_jurisdictions
-        ON ST_DWITHIN(locations.geometry, geo_jurisdictions.geometry, 150.0 / 111111.0)
+        ON public.ST_DWITHIN(locations.geometry, geo_jurisdictions.geometry, 150.0 / 111111.0)
     LEFT JOIN jurisdictions
         AS jurisdictions
         ON jurisdictions.id = locations.jurisdiction_id
```

Comment:
is this indexed?

Reply:
Yes, they are indexed [here (id)](https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/2-transaction_tables/deploy/jurisdictions.psql#L23) and [here (jurisdiction_id)](https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/2-transaction_tables/deploy/locations.psql#L39)